modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/deprecated-annotation.yaml

```diff
@@ -2,7 +2,7 @@
   rules:
   - alert: ExtendedMonitoringDeprecatatedAnnotation
     expr: >-
-      group by (kind,namespace,name) (d8_deprecated_legacy_annotation == 1)
+      group by (kind,namespace,name) (d8_deprecated_legacy_annotation{name!~"(d8.*|kube-basic-auth|kube-prometheus-pushgateway)", namespace!~"d8.*|kube-basic-auth|kube-prometheus-pushgateway"} == 1)
```

Comment:
Why do we need to exclude namespaces manually from the alert?
The problem is that on some namespaces, there are still deprecated annotations.

Example:
https://github.com/deckhouse/deckhouse/blob/a2ee5499ef58952e4ab38673e3b6d0603e81ab29/modules/402-ingress-nginx/templates/namespace.yaml#L8

Let's remove all deprecated annotations from our namespaces. This will also fix the alert.